src/components/tabs/README.md

```diff
@@ -390,34 +390,27 @@ the content of the `<b-tab>` until it is activated (shown), and will be un-mount
 deactivated (hidden):
 
 ```html
-<b-tabs>
-  <b-tab title="abc">
-    <!-- this content will always be mounted -->
-    <div>Tab ABC content</div>
-  </b-tab>
-  <b-tab title="xyz" lazy>
-    <!-- this content will not be mounted until this tab is shown -->
-    <!-- and will be un-mounted when hidden -->
-    <my-xyz-component></my-xyz-component>
-  </b-tab>
+<b-tabs content-class="mt-3">
+  <!-- This tabs content will always be mounted -->
+  <b-tab title="Regular tab"><b-alert show>I'm always mounted</b-alert></b-tab>
```

Comment:
The comment was inside the `<b-tab>` to enforce that the _content_ or the tab is lazy mounted, not the actual tab ;-)

Reply:
But I think the comment makes it clear enough what is meant.

Reply:
True